Drop-In/src/main/java/com/braintreepayments/api/dropin/DropInRequest.java

```diff
@@ -258,6 +283,11 @@ public int getCardholderNameStatus() {
         return mCardholderNameStatus;
     }
 
+    public boolean getDefaultVaultSetting() { return mDefaultVaultSetting; }
```

Comment:
Android Studio wraps these single line methods like this but I think I prefer adding some newlines:
```
public boolean getDefaultVaultSetting() {
        return mDefaultVaultSetting;
}
```

Same with `isSaveCardCheckBoxShown` below, and `isVaultManagerEnabled` could also be fixed too 🥂

Reply:
👍